example/_data/locales/fr.yml

```diff
@@ -0,0 +1,47 @@
+# Strings of the minima theme
+# language French
+posts: "Articles"
+subscribe: "s'abonner au"
+via_RSS: "flux RSS"
+abbreviated_weekday:
+  - Dim
+  - Lun
+  - Mar
+  - Mer
+  - Jeu
+  - Ven
+  - Sam
+full_weekday:
+  - Dimanche
+  - Lundi
+  - Mardi
+  - Mercredi
+  - Jeudi
+  - Vendredi
+  - Samedi
+abbreviated_month:
+  - jan.
+  - fév.
+  - mar.
+  - avr.
+  - mai
+  - juin
+  - juil.
+  - août
+  - sept.
+  - oct.
+  - nov.
+  - déc.
+full_month:
```

Comment:
why use lowercase names suddenly? If `en.yml` has `Jan.` | `Feb.`, `fr.yml` should also be `Jan.` | `Fev.` etc